users/autostart.rst

```diff
@@ -20,32 +20,141 @@ there are a number of easy solutions.
 Task Scheduler
 ~~~~~~~~~~~~~~
 
-#. Start the `Task Scheduler <https://en.wikipedia.org/wiki/Windows_Task_Scheduler>`__ (``taskschd.msc``).
-#. Create a New Task ("Action" menu -> "Create Task...").
-#. General Tab:
-    #. Name the task (for example 'Syncthing').
-    #. Check "Run whether user is logged on or not".
-    #. (recommended) Check "Do not store password. The task will only have access to local resources". Leave this option unchecked if you intend to store your folders on network drives and such.
-#. Triggers Tab:
-    #. Click "New...".
-    #. Set "Begin the task" to "At Startup".
-    #. (optional) Choose a delay.
-    #. Make sure Enabled is checked.
-    #. Click "OK".
-#. Actions Tab:
-    #. Click "New...".
-    #. [Action] should be set as "Start a program".
-    #. Enter the path to syncthing.exe in "Program/Script".
-    #. (optional) Enter the path to the parent folder of syncthing.exe in "Start in (optional)". This will allow you to use paths relative to this folder in Syncthing.
-    #. Click "OK".
+Task Scheduler is a built-in administrative tool, which can be used to
+start Syncthing automatically either at user log on, or at system
+startup. In both cases, Syncthing will open and stay invisible in
+background.
+
+You can launch Task Scheduler by going to ``Start Menu > Windows
+Administrative Tools`` and clicking on ``Task Scheduler``
+
+|Windows Task Scheduler Start Menu Screenshot|
+
+or by opening ``taskschd.msc`` from the Run command (``Win+R``).
+
+|Windows Task Scheduler Run Screenshot|
```

Comment:
These two screenshots seem a bit superfluous, the explanation should be fairly clear. And I'd consider opening the startmenu (or hitting win key) plus typing "task sched..." the easiest and most fool proof option (unless of course that doesn't work, but it should, right?).

Reply:
The problem with typing is that it is locale-dependent, so "task sched..." will work only in the English version of Windows. In the other language versions, you need to use the name of Task Scheduler in that language.

The Start Menu path is also language dependent, but at least it is located in the same place, so it is easy to find, even if your Windows is not in English. On the other hand, ``taskschd.msc`` works regardless of the Windows language.

When it comes to the screenshots, as with the Task name thing, I added them for completeness. They are probably not necessary, although from my experience, some people simply need visual cues to be able to navigate around. Written instructions just do not cut it :smirk:.

Reply:
Ah right, the locale - just recently faced that problem myself.

Personally I don't see how these help (the first one doesn't show where in the menu this item is and the second one is a popup with a single input field, not much to mistake there), but I am not the target audience so again: Feel free to override me if you feel it helps.

I don't disagree on the "some people" account, but I believe there's a limit. And that limit is where the hand-holding gets more time-intensive and the learning effect is too low, than just doing it for them (in this context: them asking "the family geek" or writing to the forum if they're advanced). And e.g. for the task name the field is so in your face that I'd say that limit is reached. While the added page length (more scrolling) affects the 99.999% that do not fail to find the very first input field on the dialogue.

Reply:
Yeah, the Start Menu screenshot is problematic. In order to show the location clearly, the whole Start Menu would have to be included, which is very tall, even in my virtual installation of Windows 10 Enterprise 2019 LTSC (i.e. only built-in tools and no Metro applications). That is why I cut it like that :sweat:.

As for the `Win+R` method, the screenshot is indeed unnecessary once you get the Run command window open, as there is nothing to mess up with there.

I would still like to include some kind of a helpful visual clue for those, who will be opening Task Scheduler for the fist time though... I will see what I can do here without adding superfluous screenshots.